packages/block-library/src/site-title/block.json

```diff
@@ -12,14 +12,21 @@
 		},
 		"textAlign": {
 			"type": "string"
+		},
+		"isLink": {
+			"type": "boolean",
+			"default": true
+		},
+		"linkTarget": {
+			"type": "string",
+			"default": "_self"
 		}
 	},
 	"supports": {
 		"align": [ "wide", "full" ],
 		"html": false,
 		"color": {
 			"gradients": true,
-			"text": false,
```

Comment:
Why was this line removed?

Reply:
Because when the site title is not a link, the link text color option is not available.
When the site title is plain text, the regular text color option needs to be available, and for that, it can not be set to false.

It does not need to be set to true, because it is enabled by default.